pom.xml

```diff
@@ -4,14 +4,18 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532.3</version>
+    <version>2.31-SNAPSHOT</version>
   </parent>
   <artifactId>validating-string-parameter</artifactId>
   <packaging>hpi</packaging>
   <name>Validating String Parameter Plugin</name>
-  <version>2.4-SNAPSHOT</version>
+  <version>2.5-SNAPSHOT</version>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Validating+String+Parameter+Plugin</url>
 
+  <properties>
+    <jenkins.version>2.46.2</jenkins.version>

```

Comment:
Definitely good point. I had to up this so that `hpi:run` started the correct version of Jenkins, and forgot to restore it afterwards. Did that now.